test/manifold_smooth_test.cpp

```diff
@@ -0,0 +1,221 @@
+// Copyright 2021 The Manifold Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <algorithm>
+
+#include "cross_section.h"
+#include "manifold.h"
+#include "samples.h"
+#include "test.h"
+#include "tri_dist.h"
+
+using namespace manifold;
+
+TEST(Manifold, Smooth) {
```

Comment:
Let's go with the convention of `TEST(Smooth, Tetrahedron)` - where the first name refers to the testing file (smooth_test.cpp) and the second is more specific. These tests have grown over time, so some of the older ones have pretty vague names we should update.

Reply:
Okay Done